Error creating new block app

I'm trying to create a new block app, with the instructions [on this page](https://blockprotocol.org/docs/developing-blocks#install) [edit: added link]. I use the following commands:

```
npx create-block-app test-block
cd test-block
yarn install
```

The `yarn install` command fails with the following error:

```
[3/4] ⠈ mock-block-dock
error /Users/austin/projects/test-block/node_modules/mock-block-dock: Command failed.
Exit code: 1
Command: yarn build
Arguments:
Directory: /Users/austin/projects/test-block/node_modules/mock-block-dock
Output:
yarn run v1.22.17
$ yarn clean; yarn build:cjs; yarn build:esm
$ rimraf ./dist/
$ tsc --p tsconfig.build.json --module commonjs --outDir dist/cjs
error TS5058: The specified path does not exist: 'tsconfig.build.json'.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
$ tsc --p tsconfig.build.json
error TS5058: The specified path does not exist: 'tsconfig.build.json'.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I'm on macOS version 12.1 / Node v16.3.1 / npm v8.4.0 / yarn 1.22.17. Please let me know if you need any more details -- I'd really like to contribute to this project. Thanks in advance!

--- a/package.json
+++ b/package.json
@@ -23,8 +23,7 @@
     "dev": "yarn workspace @blockprotocol/site dev",
     "dev:seed-db": "yarn workspace @blockprotocol/site dev:seed-db",
     "dev:db": "yarn workspace @blockprotocol/site dev:db",
-    "postinstall": "patch-package",
-    "prepare": "husky install"
+    "prepare": "husky install && patch-package && yarn workspace mock-block-dock run prepare"
   },
   "lint-staged": {
     "**/*": [

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "dev": "yarn workspace @blockprotocol/site dev",
     "dev:seed-db": "yarn workspace @blockprotocol/site dev:seed-db",
     "dev:db": "yarn workspace @blockprotocol/site dev:db",
-    "prepare": "husky install && patch-package && yarn workspace mock-block-dock run prepare"
+    "prepare": "patch-package && husky install && yarn workspace mock-block-dock run prepare"
   },
   "lint-staged": {
     "**/*": [

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "dev": "yarn workspace @blockprotocol/site dev",
     "dev:seed-db": "yarn workspace @blockprotocol/site dev:seed-db",
     "dev:db": "yarn workspace @blockprotocol/site dev:db",
-    "prepare": "patch-package && husky install && yarn workspace mock-block-dock run prepare"
+    "prepare": "patch-package && husky install && yarn workspace mock-block-dock run prepare && yarn workspace @blockprotocol/site run prepare"
   },
   "lint-staged": {
     "**/*": [

--- a/packages/block-template/README.md
+++ b/packages/block-template/README.md
@@ -4,7 +4,7 @@
 
 See https://blockprotocol.org/docs/developing-blocks
 
-**TL;DR:** Run `npx create-block-app \[your-block-name\]`
+**TL;DR:** Run `npx create-block-app [your-block-name]`
 
 ## Step two: write and build a component
 

--- a/packages/block-template/package.json
+++ b/packages/block-template/package.json
@@ -46,7 +46,7 @@
     "core-js": "^2.6.12",
     "cross-env": "^7.0.3",
     "html-webpack-plugin": "^4.5.2",
-    "mock-block-dock": "0.0.5",
+    "mock-block-dock": "0.0.4",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "regenerator-runtime": "^0.13.9",

--- a/packages/block-template/package.json
+++ b/packages/block-template/package.json
@@ -1,6 +1,6 @@
 {
   "name": "block-template",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "description": "Block template",
   "keywords": [
     "blockprotocol",
@@ -46,7 +46,7 @@
     "core-js": "^2.6.12",
     "cross-env": "^7.0.3",
     "html-webpack-plugin": "^4.5.2",
-    "mock-block-dock": "0.0.4",
+    "mock-block-dock": "0.0.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "regenerator-runtime": "^0.13.9",

--- a/packages/mock-block-dock/package.json
+++ b/packages/mock-block-dock/package.json
@@ -1,6 +1,6 @@
 {
   "name": "mock-block-dock",
-  "version": "0.0.4",
+  "version": "0.0.5",
   "description": "A mock embedding application for Block Protocol blocks",
   "keywords": [
     "blockprotocol",
@@ -31,8 +31,8 @@
     "build:esm": "tsc --p tsconfig.build.json",
     "clean": "rimraf ./dist/",
     "dev": "concurrently -n webpack,webpack-dev-server -c green,cyan \"yarn build:dev -- --watch --verbose\" \"yarn run-dev-server\"",
-    "postinstall": "yarn build",
     "lint:tsc": "tsc --noEmit",
+    "prepare": "yarn build",
     "run-dev-server": "cross-env NODE_ENV=development webpack-dev-server -d --port 9090 --open --config dev/webpack.config.js"
   },
   "dependencies": {

--- a/packages/mock-block-dock/package.json
+++ b/packages/mock-block-dock/package.json
@@ -1,6 +1,6 @@
 {
   "name": "mock-block-dock",
-  "version": "0.0.5",
+  "version": "0.0.4",
   "description": "A mock embedding application for Block Protocol blocks",
   "keywords": [
     "blockprotocol",

--- a/site/package.json
+++ b/site/package.json
@@ -13,8 +13,8 @@
     "dev:seed-db": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/seed-db",
     "generate-blocks-data": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/generate-blocks-data",
     "generate-sitemap": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/generate-sitemap",
-    "postinstall": "patch-package && yarn generate-sitemap && yarn generate-blocks-data",
     "lint:tsc": "tsc --noEmit",
+    "prepare": "patch-package && yarn generate-sitemap && yarn generate-blocks-data",
     "start": "next start"
   },
   "dependencies": {

